go-dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.17-rc-buster
+WORKDIR /casdoor
+COPY ./ /casdoor
+RUN go env -w CGO_ENABLED=0 GOPROXY=https://goproxy.io,direct GOOS=linux GOARCH=amd64 && apt update && apt install sudo && wget https://nodejs.org/dist/v12.22.0/node-v12.22.0-linux-x64.tar.gz && sudo tar xf node-v12.22.0-linux-x64.tar.gz && sudo apt install wait-for-it
+ENV PATH=$PATH:/casdoor/node-v12.22.0-linux-x64/bin
```

Comment:
I think if there are multiple commands should be displayed in separate lines to increase readability, like this:
```shell
RUN commandA &&
         commandB &&
         commandC &&
         and etc...
```